index.js

```diff
@@ -12,3 +11,36 @@ alert("Hello from index.js!");
 //Show correct costs for the level
 
 //Main task to validation rules such as ensure that the input is valid. Email check, selected hours are correct.
+
+// Display a form to the user
+// Include input fields for the user's email address, level, and hours of study
+
+//Present  check
+// Validation that hours are correct for the level
+
+// Capture user's input on form submission
+let form = document.querySelector("form");
+
+form.addEventListener("submit", function (event) {
+  event.preventDefault();
+  console.log("Form submitted");
+});
+// Store the user's email address as userEmail (string/text)
+// Store the user's level as userLevel (string/text)
+// Store the user's hours of study as userHours (number)
+// Validate the user's input
+// Check if the user has selected a level
+// Check if the user has provided an email address
+// Check if the user has specified at least one hour of study
+// Check if the number of hours requested is within the allowed range
+// Calculate the total cost
+// Display the total cost to the user
+let userName = string;
+let userLevel = string;
+let userEmail = string;
+let userHours = int;
```

Comment:
These are not quite correct - delete them for now:

- The variables themselves need to be declared inside the event handler itself
- They need to be assigned to the actual values of the input boxes. We will be looking at how to do this in class.